source/docs/03-metadata.xml

```diff
@@ -1625,6 +1625,43 @@
+                        </p>
+                        <p>This entry could be encoded as follows:</p>
+                        <p>
+                           <figure>
+                              <head/>
+                              <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample089.txt" parse="text"/></egXML>
+                           </figure>
+                        </p>
+                        <p>A slightly more structured form would be:</p>
+                        <p>
+                           <figure>
+                              <head/>
+                              <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample090.txt" parse="text"/></egXML>
+                           </figure>
+                        </p>
+                        <p>However, although a standardization of these descriptions is difficult to realize due to different practices, these transcriptions – as in the musical text – can be marked by means of <gi scheme="MEI">add</gi>, <gi scheme="MEI">del</gi>, (see module MEI.edittrans) etc. (see <ptr target="#editTrans"/>) and assigned to a specific scribe by <att>hand</att> (see <gi scheme="MEI">handList</gi>):</p>
+                        <p>
+                           <figure>
+                              <head/>
+                              <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample091.txt" parse="text"/></egXML>
+                           </figure>
+                        </p>
+                        <p>For structuring purposes, it may sometimes be useful to separate entries made by a composer in the manuscript from those made by others:</p>
+                        <p>
+                           <figure>
+                              <head/>
+                              <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample092.txt" parse="text"/></egXML>
+                           </figure>
+                        </p>
+                        <p>Depending on how one defines entries, <gi scheme="MEI">stamps</gi> can also be encoded under <gi scheme="MEI">addDesc</gi>.
```

Comment:
```suggestion
                        <p>Under certain circumstances, <gi scheme="MEI">stamps</gi> can also be encoded under <gi scheme="MEI">addDesc</gi>.
```

It would probably be useful to provide an example of what these circumstances are... I suggested this change because you also don't try to define "entries", even though you say that it depends on how you define it. Using this wording it frees you from needing a long definition, and then you can just provide a "for example..."

Reply:
@gucl-mu Could you have a look at this, please?

Reply:
The element is `<stamp>` not `<stamps>`

Reply:
Sorry for late replay, could one of theese two can solve the problem: "Depending on how much is subsumed under the term entries" or "Depending on how broad the definition of entries is choosen," ->My intention was to say that the user/encoder decides whether to encode stamps under `<addDesc>` or separately. 
Would `<gi scheme="MEI">stamp</gi>`**s** also be possible?

Reply:
Well, you actually don't specify why the `<stamp>` element can be used. What does the element do? What does it record? Telling the reader what the `<stamp>` element does is, in my mind, more important than just saying it can be used.

I wouldn't really recommend your suggestion for pluralizing the `<stamp>` element, since it would be rendered as `<stamp>s` which just looks weird. You could say something like "Stamps can be recorded using multiple `<gi scheme="MEI">stamp</gi>` elements." if you really wanted to emphasize the plural nature of it.

Reply:
ah ok, thanks for your comment, now I get it. I had written a separate description for stamps, should we maybe just refer to it here (and @riedde  is this description already added somewhere?).

Reply:
And you are right `<stamp>s` looks weird.

Reply:
@gucl-mu Yes, the new stamp section is in Pull Request #1095